python/orca/src/bigdl/orca/learn/pytorch/experimential/mmcv/mmcv_ray_runner.py

```diff
@@ -198,11 +198,19 @@ def validate(self, **kwargs):
 
     def get_state_dict(self):
         """Returns the state of the runner."""
-        pass
+        state = {
+            "epoch": self._epoch,
+            "model": self.model.module.state_dict(),
+            "optimizer": self.optimizer.state_dict()
+        }
+        return state
 
     def load_state_dict(self, state):
         """Sets the state of the model."""
-        pass
+        self.model.module.load_state_dict(state["model"])
```

Comment:
Then I believe our `save_checkpoint` also needs to be fixed as model.module. Can we keep self.model not in DDP mode?